### ISSUE TITLE
Page Rename

### ISSUE THREAD

**Issue by Almi14** [turnermm/numbering#10]
Hi there, 

if i page is renamed the Link with the ID will no longer work. 
Is there anything planed to solve it?

Cheers Markus

**Comment by turnermm** [comment]
The Numbering plugin doesn 't retain page id  information.

**Comment by Almi14** [comment]
Ok but is there a known solution, for that? We had the hope with that plugin we don´t care about Page renaming.

**Comment by turnermm** [comment]
I don't think you've explained the problem fully enough.  Do you mean the numbering output does not keep up with the page change, so that if you h ave apge id in the output  it is no logner correct?  Or do you mean that the numbering output disappears when the page is moved?

**Comment by Almi14** [comment]
Sorry, so what i mean is "...Or do you mean that the numbering output disappears when the page is moved?"

Thanks!

**Comment by turnermm** [comment]
I don't understand how that could happen.  Once the number is placed on your page, it's there just as though you had typed it in yourself.  It doesn't matter that the page has been renamed.  If you embed the id of the current page into the output, the numbering plugin does not keep track of that--it keeps track only of the number.

**Comment by Almi14** [comment]
Hi,
sorry i think i didn´t express me correct. I need more to explain, i don´t talk about the output, only the url and the need that the page reachable, no matter it´s renamed or not, through it´s id. For example:

A page´s url is https://dokuwiki.com/doku.php?id=namespace:site1 and the id url is https://dokuwiki.com/id.php?id=00532 

If i rename the page to https://dokuwiki.com/doku.php?id=namespace:site2 i´m not longer able to use the id URL 

Is this expected or do you know a work around?

Thanks!

**Comment by turnermm** [comment]
First, I am not familiar with this way of  referring to pages by two different types urls, page and id.  Dokuwiki always refers to the id as `namespace:pagename`.  So, it's possible that you have another plugin which is creating what you call the `id`.  

Second, I can't see how the numbering plugin can affect this scheme.  It merely places a string of text into your page.

**Comment by Almi14** [comment]
Hi, 

i´m little bit confused. 

I thought that your plugin make a articel always avaible thru the ID which is shown by a mouse over s. the picture.

There´s no other plugin which is creating an id.

Cheers Markus
![mouseover](https://user-images.githubusercontent.com/26408723/28362154-b55f4958-6c7b-11e7-966c-a9d6f0110532.png)

**Comment by turnermm** [comment]
No.  All it does is to insert numbers in sequence into your page.

**Comment by Almi14** [comment]
Ok so it would be a feature request :)
